sysutils/mise/Portfile

```diff
@@ -86,6 +86,7 @@ notes {
 }
 
 
+
```

Comment:
We're adding extra empty space - is this intentional?

Reply:
No. I'm not quite sure why it's showing up (I adjusted the tooling I use for prepping PRs with `cargo.crates` and I may have accidentally added an extra blank line).